OpenEdXMobile/src/test/java/org/edx/mobile/view/CourseUnitYoutubeVideoFragmentTest.java

```diff
@@ -0,0 +1,134 @@
+package org.edx.mobile.view;
+
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import org.edx.mobile.R;
+import org.edx.mobile.course.CourseAPI;
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+import org.edx.mobile.model.course.CourseComponent;
+import org.edx.mobile.model.course.CourseStructureV1Model;
+import org.edx.mobile.model.course.VideoBlockModel;
+import org.edx.mobile.util.VideoUtil;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.robolectric.shadows.support.v4.SupportFragmentTestUtil;
+
+import java.io.IOException;
+
+import roboguice.activity.RoboFragmentActivity;
+
+import static org.edx.mobile.http.util.CallUtil.executeStrict;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+
+public class CourseUnitYoutubeVideoFragmentTest extends UiTest {
+
+
+    private static final String YOUTUBE_VIDEO = "YOUTUBE_VIDEO"; // Config key for YOUTUBE
+
+    @Mock
+    VideoUtil videoUtil;
```

Comment:
`videoUtil` unused field needs to remove.

Reply:
removed